inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+Each layer provides appropriate fields, so called *features* to enter a lable as what to annotate the selected text part. For example, on the _Named Entity_ layer in {product-name}, you find two feature-fields: _value_ and _identifier_. In _value_, you can enter what kind of entity it is ("LOC" for a location, "PER" for a person, "ORG" for an organization and "OTH" for other) In _identifier_ you can enter which concrete entity (which must be in the knowledge base) it is. For the example "Paris", ot may be the French capital; the person Paris Hilton; a company named "Paris" or something else. +
+{product-name} provides default layers with default features to give you a starting point. Default layers cannot be deleted as selfmade layers can. However, new features can be added.
+See the main documentation for details on <<Layers>>, features, the different types of layers and features, how to create custom layers and how to adjust them for your individual task.
+
+* *Layers:* In this section, you may modify existing layers to use for
+your annotations and also create own layers. In case you do not want to work
+on default layers only but wish to use custom layers designed for your individual task, we recommend reading the documentation for details on <<Layers>>.
+
+
+[[tagsets_in_getting_started]]
+NOTE: *Tagsets:* In order all annotations to have consistent labels, it is preferrable to use defined tags which can be given to the annotations. If users do not enter free text for a lable but stick to predefindes tags, they avoid different names for the same thing and varying spelling. A set of such definded tags is called a tagset. It is so to say a collection of lables which can be used for an annotation. {product-name} comes with predefined tagsets out of the box and like many times before, they serve as a suggestion and starting point only. You can modify them or create your own ones.
+
+NOTE: *Feature Types:* The tags of your tagsets must always fit the feature type of the feature they are used for. The feature type defines what type of information the feature can be, for example "Primitive: Integer" for whole numbers, "Primitive: Float" for decimals; "Primitive: Boolean" for a true/false lable only; the most common one "Primitive: String" for text lables or "KB: Concept/Instance/Property" if the feature shall link to a knowledge base. There are more types for features but these are the most important ones for you to know. +
+Changing the type does only work for your own features, not for default features. In order to do so, scroll in the "Feature Details" panel until you see the field "Type" and select the type of your choice.
+If a tagset shall be linked to a feature, they must have the same type.
+For more details, see the <<Features>> in the main documentation.
+
+* *Tagsets:* Behind this tab, you can modify and create the tagsets for your layers. Tagsets are always bound to a layer, or more precisely to a certain feature of a layer.
```

Comment:
What are the reasons for your preference?
I see it's not so nice in this case, especially because it's two boxes before the actual topic "Tagsets" starts. But it's the least bad decision in my opinion.

The reasons for my decision are: 
- It should be consistent (always before or always after the term is mentioned)
- Both decisions bring problems with them
- Its better to know a term before it is used, so I put the box above
- Boxes destroy the levels of bulletpoints. so you cannot put a box between bulltpoint with level one and a subordinate bulletpoint
- Putting the box on tagsets on the very end of the "tagset"-bulletpoint (with all its subodinated bulletoints), the explanation is too far away from the first mention.
- putting one of the two above and one below the tagset-bulletpoint (with all the subordinated ones) is bad bacause box on feature types explaines the mention of feature types in the box on tagsets. we cannot tear them apart.